src/main/java/mods/railcraft/common/gui/containers/RailcraftContainer.java

```diff
@@ -66,7 +66,7 @@ public void addListener(IContainerListener listener) {
     @Override
     public final void detectAndSendChanges() {
         super.detectAndSendChanges();
-        if (FMLCommonHandler.instance().getEffectiveSide() == Side.SERVER) {
+        if (FMLCommonHandler.instance().getSide() == Side.SERVER) {
```

Comment:
This breaks everything. These functions have vastly different meanings. I need to know which thread I'm on, not which binary.

Reply:
Will revert this one.